modules/tinymce/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Fixed
+- Flash of unstyled content while loading the content css was shown. It will now load the content css first then populate the editor #TINY-7249
```

Comment:
If we're going ahead with this, then this really needs to be a `Changed` entry as well. We really need to call out/detail that we're changing the initialization sequence here as that could break some people integrations.

Reply:
Added a change item as well since it's a behavior change for sure. Who knows what people might be doing out there.